code/modules/clothing/clothing.dm

```diff
@@ -477,6 +477,13 @@ BLIND     // can't see anything
 			. += mutable_appearance('icons/effects/item_damage.dmi', "damaged[blood_overlay_type]")
 		if(blood_DNA)
 			. += mutable_appearance('icons/effects/blood.dmi', "[blood_overlay_type]blood")
+		var/mob/living/carbon/human/M = loc

```

Comment:
hmm, there will be a way to make it layer correctly, I'm sure of it.
Might just have to add a way for objects to pass their MAs to the mob, that way you can use a negative layer (```-ABOVE_SUIT_LAYER``` would be correct) to layer it within the mob.

for example, worn_overlays could instead return an assoc list.
```list("parent" = list(current worn overlays behaviour, go to our parent), "mob" = list(things that bypass our parent and go on the mob))```
Can even have it so that if the list isn't assoc (ie: its first element doesnt have a value (```!llist[list[1]```) then assume the old behaviour (ie: if you don't make it assoc it just works in "parent" mode)

Do you feel up to that? It's not too complex.
If not, I can do it on my lunch break (about 1 hour 30 to 2 hours from now)